### PR TITLE
Add Abhilash Sreenivasa to the roster.

### PR DESCRIPTION
**Pull request by abhilashSreenivasa**

### CODE REVIEW

File: roster.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ## Students
 
-
+-Abhilah Sreenivasa,[abhilashSreenivasa](https://github.com/abhilashSreenivasa)
```
Comment (kbuffardi) [inline roster.md:9]: There needs to be a space between the `-` and your name

Comment (abhilashSreenivasa) [reply roster.md]: Hi, It has been fixed. Thank you.

File: roster.md [hunk 2]
```diff
@@ -6,6 +6,11 @@
 - Kevin Buffardi [kbuffardi](https://github.com/kbuffardi)
 
 ## Students
+<<<<<<< HEAD
```
Comment (kbuffardi) [inline roster.md:9]: Need to get rid of the git-generated annotations of the conflict, including:

`<<<<<<< HEAD` which marks the beginning of one version
`=========` which separates the end of that version from the beginning of the next
`>>>>>>> main` which marks the end of the second version

Comment (abhilashSreenivasa) [reply roster.md]: My bad,  I think it has been fixed now.

**Pull request merged**